modules/core/src/test/scala/org/tessellation/http/routes/TrustRoutesSuite.scala

```diff
@@ -63,4 +85,67 @@ object TrustRoutesSuite extends HttpSuite {
       }
       .unsafeRunSync()
   }
+
+  test("GET trust latest succeeds") {
+    val req = GET(uri"/trust/latest")
+
+    val gen = for {
+      trustInfo <- Gen.mapOfN(4, genPeerTrustInfo)
+    } yield TrustMap.empty.copy(trust = trustInfo)
+
+    forall(gen) { trust =>
+      KryoSerializer
+        .forAsync[IO](sdkKryoRegistrar.union(coreKryoRegistrar))
+        .use { implicit kryoPool =>
+          for {
+            ts <- mkTrustStorage(trust)
+            gossip = new Gossip[IO] {
+              override def spread[A: TypeTag: Encoder](rumorContent: A): IO[Unit] = IO.unit
+
+              override def spreadCommon[A: TypeTag: Encoder](rumorContent: A): IO[Unit] = IO.unit
+            }
+            tp = TrustPush.make[IO](ts, gossip)
+            routes = TrustRoutes[IO](ts, tp).p2pRoutes
+
+            expected = TrustScores(
+              trust.trust.view
+                .mapValues(_.predictedTrust)
+                .collect { case (k, Some(v)) => (k, v) }
+                .toMap
+            )
+          } yield expectKryoBodyAndHttpStatus(routes, req)(expected, Status.Ok)
+        }
+        .unsafeRunSync()
```

Comment:
Same comment as earlier. You don't need this if you follow the suggestion above.

Reply:
Fixed.